src/sum.js

```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
+  return a + b;
+}
```

Comment:
This one too can be refactored as ES6 arrow function, that is 
`let sum = (a, b) => a + b`

Reply:
Done